h2o-r/tests/testdir_jira/runit_pubdev_6475.R

```diff
@@ -0,0 +1,22 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
+source("../../scripts/h2o-r-test-setup.R")
+
+
+
+test.tree.rootnode_only <- function() {
+  
+  data <- h2o.importFile(locate("smalldata/iris/iris_wheader.csv"))
+  model <- h2o.gbm(x= c("sepal_len", "sepal_wid"), y = "class", training_frame = data)
+  # Artificially create an XGBoost model with root node only
+  model <- h2o.xgboost(x= c("sepal_len", "sepal_wid"), y = "class", training_frame = data, max_depth = 0)
```

Comment:
h2o.xgboost accepts max_depth = 0.  And XGBoost accepts the parameter for some reason. But it is useful for testing this scenario.